starters/gatsby-starter-mdx-basic/README.md

```diff
@@ -0,0 +1,27 @@
+# Gatsby Starter MDX Basic
+
+View this starter [live on Netlify](https://gatsby-starter-mdx-basic.netlify.com/)
```

Comment:
Is this still the URL from chris or ours?

Reply:
Chris. I was going to keep it and then I can redeploy? Or I could just remove it.

Reply:
That's probably on his account so you can't change the source? 🤔 Then just remove it for now